recipes/util-log-to-console-log/src/workflow.ts

```diff
@@ -0,0 +1,98 @@
+import type {
+	Edit,
+	Kinds,
+	Range,
+	Rule,
+	SgNode,
+	SgRoot,
+	TypesMap,
+} from "@codemod.com/jssg-types/main";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import { resolveBindingPath } from "@nodejs/codemod-utils/ast-grep/resolve-binding-path";
+import { removeBinding } from "@nodejs/codemod-utils/ast-grep/remove-binding";
+import { removeLines } from "@nodejs/codemod-utils/ast-grep/remove-lines";
+
+type BindingToReplace = {
+	rule: Rule<TypesMap>;
+	node: SgNode<TypesMap, Kinds<TypesMap>>;
+	binding: string;
+};
+
+/*
+ * Transforms `util.log($$$ARG)` usage to `console.log(new Date().toLocaleString(), $$$ARG)`.
+ *
+ * Steps:
+ *
+ * Locate all util.log imports, noting the replacement rule, import node, and binding name.
+ *
+ * For each binding, replace calls to util.log($$$ARG) with the new console.log format,
+ * and determine if the import line should be updated or removed.
+ *
+ * Apply all changes, removing or updating the import line as needed.
+ */
+export default function transform(root: SgRoot): string | null {
+	const rootNode = root.root();
+	const edits: Edit[] = [];
+	const linesToRemove: Range[] = [];
+	const bindsToReplace: BindingToReplace[] = [];
+
+	const nodeRequires = getNodeRequireCalls(root, "util");
+	const nodeImports = getNodeImportStatements(root, "util");
+	const path = "$.log";
+	const importRequireStatement = [...nodeRequires, ...nodeImports];
+
```

Comment:
```suggestion

	if (!importRequireStatement.lenght()) return null;

```

If not imported it's mean that we won't touch to the file